docs/using/alerting.md

```diff
@@ -170,8 +170,10 @@ To silence alerts:
 To reactivate silenced alerts, click **Unsilence All** or click on the disabled bell icon <i class="uil uil-bell-slash"></i> corresponding to the alerts you want to reactivate.
 
 ## Alternative alerting options
-PMM Integrated Alerting was introduced as an alternative to Grafana Alerting and Prometheus’ Alertmanager.
+| Use | <i class="uil uil-thumbs-up"></i> **Benefits** | <i class="uil uil-thumbs-down"></i> **Drawbacks**
+|---|---|---
+| [External Alertmanager](/how-to/configure.html#alertmanager-integration) | 1. A way to integrate PMM when a company has an existing [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/). | 1.  For experts who use it already 
+| **Recommended**<br>[Native Grafana Alerting](https://grafana.com/docs/grafana/v8.5/alerting/) | 1. Build-in<br>2. Feature-rich. | 1. Available in the latest PMM versions only.
```

Comment:
```suggestion
| [Native Grafana Alerting](https://grafana.com/docs/grafana/v8.5/alerting/) | 1. Built-in<br>2. Feature-rich. | Available in the latest PMM versions only.
```

Reply:
Is it a good idea to recommend Native Grafana Alerting as we want to promote Integrated Alerting...

Reply:
no,   in 2.31 we;ll merge  Integrated Alerting and Native Grafana alerting and  -  Native Grafana alerting  will be the base for this feature